addons/compat_rhs_usf3/XEH_postInit.sqf

```diff
@@ -0,0 +1,31 @@
+#include "script_component.hpp"
+
+// Don't duplicate the event
+if ("ace_compat_rhs_afrf3_arsenal" call EFUNC(common,isModLoaded)) exitWith {};
+
+#define ITEM_INDEX_SIDE 1
+#define ITEM_INDEX_OPTIC 2
+#define ITEM_INDEX_BIPOD 3
+
+// Compatibility for attachments
+// Will only work for players or RC unit, different arsenal centers will be ignored
+[QEGVAR(arsenal,weaponItemChanged), {
+    params ["_weapon", "_item", "_itemIndex"];
+    if (EGVAR(arsenal,center) != ACE_player) exitWith {};
+
+    switch (_itemIndex) do {
+        case ITEM_INDEX_SIDE: {
+            call rhs_fnc_anpeq15_rail;
+        };
+        case ITEM_INDEX_OPTIC: {
+            [] call rhs_fnc_preferredOptic;
```

Comment:
with just rhsUS it's undefined but there is

```suggestion
            [] call RHS_fnc_preferedOptic
```
not sure

Reply:
I forgot to remove this bit. Optic handling isn't necessary as it'll get caught by `baseWeapon` and switched back by RHS on arsenal closing anyway.